dolphinscheduler-master/src/test/java/org/apache/dolphinscheduler/server/master/integration/cases/WorkflowStartTestCase.java

```diff
@@ -141,15 +145,32 @@ public void testStartWorkflow_with_oneSuccessTaskUsingTaskGroup() {
         workflowOperator.manualTriggerWorkflow(workflowTriggerDTO);
         workflowOperator.manualTriggerWorkflow(workflowTriggerDTO);
 
+        final TaskDefinition taskDefinition = context.getTasks().get(0);
+        final Map<Integer, TaskGroupQueue> taskGroupQueueMap = new HashMap<>();
+
         await()
                 .atMost(Duration.ofMinutes(2))
                 .atLeast(Duration.ofSeconds(20))
                 .untilAsserted(() -> {
+
+                    // Capture TaskGroupQueue records during execution since they are deleted after task completion.
+                    repository.queryAllInQueueTaskGroupQueue().forEach(taskGroupQueue -> {
+                        taskGroupQueueMap.put(taskGroupQueue.getId(), taskGroupQueue);
+                    });
```

Comment:
It's better to remove this assertion, this might easily failure. If we want to do this assertion we should avoid  delete this in db in ci.

Reply:
I think you should remove these code related to `taskGroupQueueMap` and remove the method `queryAllInQueueTaskGroupQueue`, this PR can be merged.